src/App/Kernel.php

```diff
@@ -473,7 +477,12 @@ public function validate(Request $request, Response $response)
             }
         } catch (Throwable $e) {
             if ($this->app['config']['debug']) {
-                throw $e;
+                $json = json_decode($response->getContent());
+                $json->_validationInfo = $e->getMessage();
+                $json->_time = microtime(true) - $this->startTime;
+                $response->setContent(json_encode($json));
+
+                return $response;
```

Comment:
This horrible little snippet for debugging allows you to see the validation errors inside of the invalid JSON in debug to catch potential issues